index.js

```diff
@@ -125,7 +127,30 @@ roombaAccessory.prototype = {
                 }
             });
         }
-    },
+               else {
+        this.log("Roomba Pausing");
```

Comment:
Let's change to "Roomba pausing" to match the sentence case style of other logs. However let's discuss the approach for achieving this before going further here... as we say we're pausing, but actually we're stopping.

Reply:
I've changed to "Roomba is stopping" to be more accurate and consistent for now.